ui/packages/components/src/RunsPage/RunsPage.tsx

```diff
@@ -356,11 +373,9 @@ export function RunsPage({
               />
             </div>
             <CodeSearch
-              onSearch={(content) => {
-                scrollToTop();
-                onSearch(content);
-              }}
+              onSearch={onSearchChange}
```

Comment:
Does this need to be url encoded or is that already happening in the search component?

Reply:
In theory the URLSearchParams API handles coding and decoding. In case we spot any edge cases not covered by it, I can add another layer of encoding to it